guide/using-numerical-optimization-methods.md

```diff
@@ -224,7 +224,7 @@ parameter evaluation. When the robot controller finishes the evaluation, it
 writes the fitness result into another text file and then it calls the
 `wb_supervisor_simulation_quit()` function to terminate Webots. Then the control
 flow returns to the optimization program that can read the resulting fitness,
-associate it with the current genotype and proceed with the next genotype.
+associated it with the current genotype and proceed with the next genotype.
```

Comment:
I'm not sure about this modification because the optimization program does three actions: read the fitness, associate the fitness, and proceed.

Maybe if it is not clear we could rephrase the sentence or replace ``it`` with ``fitness``.